nuxeo-runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/model/impl/ExtensionPointImpl.java

```diff
@@ -102,7 +103,13 @@ public Extension createExtension(Element element) {
                     }
                 }
             }
-            contribs = xmap.loadAll(new XMapContext(extension.getContext()), extension.getElement());
+            try {
+                contribs = xmap.loadAll(new XMapContext(extension.getContext()), extension.getElement());
+            } catch (XMapException e) {
+                throw new RuntimeException(
+                        e.getMessage() + " while processing component: " + extension.getComponent().getName().getName(),
+                        e);
```

Comment:
In practice for now we only have one thrower of `XMapException`. Also multi-line error messages are BAD for grep and file processing. Length is not an issue.